packages/amplify-java-function-runtime-provider/resources/localinvoke/src/main/java/shim/Invoker.java

```diff
@@ -36,7 +37,8 @@ private Invoker(final String lambdaJarPath, final String handlerClassName, final
                 .filter(method -> method.getName().equals(handlerMethodName))
                 .findFirst()
                 .orElseThrow(() -> new IllegalStateException("Could not find handler method named " + handlerMethodName + " in class " + handlerClassName));
-        handlerInstance = classToLoad.newInstance();
+        Constructor<Class<?>> cons = classToLoad.getConstructor();
```

Comment:
nit: final and java convention would be to name the variable 'constructor'

Reply:
done!!